calculator.py

```diff
@@ -1,11 +1,54 @@
 def addition(number_1, number_2):
+    return number_1 + number_2
 
 def subtraction(number_1, number_2):
+    return number_1 - number_2
 
 def multiplication(number_1, number_2):
+    return number_1 * number_2
 
 def division(number_1, number_2):
+    if(number_2 ==0):
+        return None
+    else:
+        return number_1 / number_2
 
 def get_input():
+    
+     try:
+    
+        number_1=float(input("Enter first number : "))
+        number_2=float(input("Enter second number: "))
+        operator= input("select the opertaion to be performed: '+','-','*','/' ")  
 
-input()
\ No newline at end of file
+     except:
+         print("Enter a valid number or operator")
+
+     else: 
+         if( operator =='+'):
+             print(number_1,"+",number_2, "=", addition(number_1,number_2))
+             
+         elif(operator == '-'):
+              print(number_1,"-",number_2, "=", subtraction(number_1,number_2))
+              
+         elif(operator == '*'):
+             print(number_1,"*",number_2,"=",multiplication(number_1,number_2))
+             
+         elif(operator == '/'):
+             answer = division(number_1,number_2)
```

Comment:
answer is not a proper name. Kindly use a name like
division_result

Reply:
ok mam